src/stories/containers/ActorProjects/components/DeliverableViewModeToggle/DeliverableViewModeToggle.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from '@emotion/styled';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import React from 'react';
+import type { DeliverableViewMode } from '../ProjectCard/ProjectCard';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
+
+interface DeliverableViewModeToggleProps {
+  deliverableViewMode: DeliverableViewMode;
+  onChangeDeliverableViewMode: (deliverableViewMode: DeliverableViewMode) => void;
+}
+
+const DeliverableViewModeToggle: React.FC<DeliverableViewModeToggleProps> = ({
+  deliverableViewMode,
+  onChangeDeliverableViewMode,
+}) => {
+  const { isLight } = useThemeContext();
+
+  return (
+    <ViewModeContainer>
+      <Button
+        isLight={isLight}
+        active={deliverableViewMode === 'compacted'}
+        onClick={() => onChangeDeliverableViewMode('compacted')}
+      >
+        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path d="M3 3.6C3 3.26863 3.26863 3 3.6 3H9.9C10.2314 3 10.5 3.26863 10.5 3.6V9.9C10.5 10.2314 10.2314 10.5 9.9 10.5H3.6C3.26863 10.5 3 10.2314 3 9.9V3.6Z" />
+          <path d="M13.5 3.6C13.5 3.26863 13.7686 3 14.1 3H20.4C20.7314 3 21 3.26863 21 3.6V9.9C21 10.2314 20.7314 10.5 20.4 10.5H14.1C13.7686 10.5 13.5 10.2314 13.5 9.9V3.6Z" />
+          <path d="M3 14.1C3 13.7686 3.26863 13.5 3.6 13.5H9.9C10.2314 13.5 10.5 13.7686 10.5 14.1V20.4C10.5 20.7314 10.2314 21 9.9 21H3.6C3.26863 21 3 20.7314 3 20.4V14.1Z" />
+          <path d="M13.5 14.1C13.5 13.7686 13.7686 13.5 14.1 13.5H20.4C20.7314 13.5 21 13.7686 21 14.1V20.4C21 20.7314 20.7314 21 20.4 21H14.1C13.7686 21 13.5 20.7314 13.5 20.4V14.1Z" />
+        </svg>
+      </Button>
+      <Button
+        isLight={isLight}
+        active={deliverableViewMode === 'detailed'}
+        onClick={() => onChangeDeliverableViewMode('detailed')}
+      >
+        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
It's any particular reason to use SVG this way and not as component

Reply:
Those SVG are not intended to be reused, so there's not need to create a new component for those cases